.gitignore

```diff
@@ -120,3 +120,4 @@ modules.xml
 
 go.work*
 .env
+.DS_Store
```

Comment:
Should probably just add this to your dev machine as a global gitignore (similar to .idea, .swp files, etc)